File: JavaScript/1019.js
// Problema: 1019 - Conversão de Tempo | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s]   | Tempo: 0.029s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');

function div(valor, divisor) {
    let qtde = parseInt(valor / divisor);
    let resto = valor % divisor;
    return [qtde, resto];
}

var s = parseInt(input.split('\n')),
    m, h;

[h, s] = div(s, 3600);
[m, s] = div(s, 60);

console.log("%d:%d:%d", h, m, s);

File: Java/1019.java
// Problema: 1019 - Conversão de Tempo | Resposta: Accepted
// Linguagem: Java 19 [+2s]            | Tempo: 0.096s

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	
	public static int[] divisor(int valor, int divisor) {
		int n = (valor / divisor),
			r = valor % divisor;
		return new int[] { n, r };
	}

    public static void main(String[] args) throws IOException {
	    BufferedReader in = 
	    	new BufferedReader(new InputStreamReader(System.in));
	    
	    int h, m, s = Integer.parseInt(in.readLine());
	    int[] ret;
	    
	    ret = divisor(s, 60*60);
	    h = ret[0]; s = ret[1];
	    ret = divisor(s, 60);
	    m = ret[0]; s = ret[1];
	    
	    System.out.printf("%d:%d:%d\n", h, m, s);
    }
}